measurement/plugins/ip_route/results_py3.py

```diff
@@ -7,8 +7,7 @@
 
 @dataclass(frozen=True)
 class IPRouteMeasurementResult(MeasurementResult):
-    """Encapsulates the result from a IPRoute measurement.
-    """
+    """Encapsulates the result from a IPRoute measurement."""
```

Comment:
_an_ IPRoute measurement...

sorry

Reply:
I think this one is actually correct (for once) :laughing: this change was due to a `black` update I believe